messaging/src/androidTest/java/io/lantern/messaging/MessagingTest.kt

```diff
@@ -116,8 +116,9 @@ class MessagingTest : BaseMessagingTest() {
                                 dog.db.listPaths(Schema.PATH_CONTACT_MESSAGES.path("%")).count()
                             )
                             assertEquals(
-                                0,
-                                dog.db.listPaths(Schema.PATH_CONTACTS.path("%")).count()
+                                1,
```

Comment:
This was update because now we are handling a permanent contact 'myself' so we have 1 contact instead of 0